res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1171,6 +1286,7 @@ TraktorS4MK2.callbackLoopSize = function(field) {
       engine.setValue(field.group, "loop_halve", 0);
     }
   }
+  // TraktorS4MK2.sendLoopSizeMessage(group, engine.getValue(field.group, "beatloop_size"));
```

Comment:
Connect changes in `beatloop_size` to an [output callback function](https://mixxx.org/wiki/doku.php/midi_scripting#connect_output_callback_functions) instead of sending the update to the controller when input is received. The former makes sure that the controller stays in sync even if the user changes beatloop_size by clicking on screen.